src/prefect/client/orchestration.py

```diff
@@ -203,7 +203,11 @@ def __init__(
 
         if api_version is None:
             # deferred import to avoid importing the entire server unless needed
-            from prefect.server.api.server import SERVER_API_VERSION
+            try:
+                from prefect.server.api.server import SERVER_API_VERSION
+            except ImportError:
+                # support the case where server components are not available
+                SERVER_API_VERSION = "0.8.4"
```

Comment:
Let's move this up to a common spot that both the client and server can reference, that way we only have one definition for it.

Reply:
I added a new file `prefect.server.api.constants` that we'll use as the source-of-truth for server and client-side code. Server components import the version directly from this file whereas client components will import from `prefect.client.constants` - which internally tries to import from the server constants. My gut is that helps with separating client/server but let me know if that feels like over-compensation.

Reply:
I think we're still missing the important part for me, that we only have one definition of that API version constant.  Could that constants file just be in a location that both `prefect-client` and `prefect-server` include in their packages?  

One kind of non-obvious thing is that the _server_ could import it from somewhere like `prefect.client.constants` because the server is the full `prefect` package.  Otherwise somewhere in utilities might make sense.  My biggest thing is that we just have one copy of this to update.

Reply:
Summary of the sync-up: I learned that `SERVER_API_VERSION` is mostly used in the client's request headers to ensure that we're backwards compatible. So we decided to put it in `prefect.client.constants` and it should be there now!